samples/sample-04/gradle.properties

```diff
@@ -0,0 +1 @@
+kotlin.code.style=official
```

Comment:
All files must finish with blank like.
You can configure your IDE to do that for you on save.

On the other hand: why do you use `official` value for this property?
Isn't it really official by default? 😕

Reply:
Oh, it was necessary before [Kotlin 1.3](https://kotlinlang.org/docs/reference/code-style-migration-guide.html), thanks for pointing out!